doc/tools/docgram/common.edit_mlg

```diff
@@ -889,6 +873,7 @@ ltac_expr1: [
 | REPLACE reference LIST0 tactic_arg
 | WITH reference LIST1 tactic_arg
 | l1_tactic
+| DELETE binder_tactic
```

Comment:
This sounds bad, it means binder_tactic completely disappears from grammar in ltac.rst

Reply:
I did this because I had the impression that `binder_tactic` was already not described as a proper grammar in `ltac.rst`.

Previously, we had:
- already `DELETE binder_tactic` in `ltac_expr5`
- already no description of `binder_tactic` besides the note ":production:`binder_tactic`\s are: :tacn:`fun` and :tacn:`let`"

However, I agree that it should appear in `ltac_expr1`. But then,  when I remove the DELETE above, `doc_gram_rsts` gives
```
Error: Tactic not in doc: @binder_tactic
```
What am I missing?

Reply:
I pushed a commit that seems a tad better but that still not perfect. @jfehrle feel free to improve, I'm still far from mastering the docgram tool and I didn't manage to have `binder_tactic` explicitly appear in `ltac_expr1` without splicing it.